google/cloud/forseti/scanner/scanners/iap_scanner.py

```diff
@@ -288,6 +288,8 @@ def make_iap_resource(self, backend_service, project_full_name):
         alternate_services = set()
         direct_access_sources = set()
         for backend in backend_service.backends:
+            if not backend.get('group'):
+                continue
             instance_group = self.find_instance_group_by_url(
```

Comment:
Can we instead do the check in find_instance_group_by_url() ? if we return None when group is not found, it will hit the next if statement (if not instance_group) and continue as expected

Reply:
+1

Reply:
That was the first thing I had tried, but I had lint error for inconsistent return.  Do you want me to disable that?

Reply:
Okay, looks like I have to explicitly pass back a `None`, rather than just a return.  Done.